L1Trigger/L1TMuonEndCap/src/PrimitiveMatching.cc

```diff
@@ -465,13 +454,11 @@ void PrimitiveMatching::insert_hits(
       (conv_hit_i.BX()         == conv_hit_j.BX()) &&
       (conv_hit_i.Strip_low()  == conv_hit_j.Strip_low()) && // For RPC clusters
       (conv_hit_i.Strip_hi()   == conv_hit_j.Strip_hi()) &&  // For RPC clusters
-      //(conv_hit_i.Roll()       == conv_hit_j.Roll()) &&
+      (conv_hit_i.Roll()       == conv_hit_j.Roll()) &&      // For RPC clusters
```

Comment:
This should have been removed rather than commented-out, since the FW does not have access to the RPC roll.

Reply:
I see.  This is because we do not check for theta equality, since same-phi / different-theta duplicates are possible for the CSC LCTs.

Reply:
Yes. Also I'm handling the RPC hit information a little bit differently than in FW, because FW doesn't even know about strip and roll numbers. I made the change a while ago and I hope it's still correct.